src/js/index.js

```diff
@@ -11,6 +11,7 @@ import {
 } from "./utils/StateUtils";
 import { createNoteMark } from "./utils/SchemaUtils";
 import SharedNoteStateTracker from "./SharedNoteStateTracker";
+import v4 from "uuid/v4";
```

Comment:
maybe rename this import to `uuid` so line 39 is clearer

Reply:
Good idea, I've done this elsewhere as well.